src/components/Search.js

```diff
@@ -6,26 +6,29 @@ class Search extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      // default placesholders while page is loading
-      location: 'Enter Location',
-      date: 'mm/dd/yyyy',
+      location: 'Enter a location',
     };
   }
 
   componentDidMount() {
-    // sets placeholder values of the date & location fields to curr loc and curr date
-    const today = getdate('mm/dd/yyyy');
-    const _this = this;
+    // sets current location for on page load
```

Comment:
This is looking good as I test it! I had no problems in searching for New York City or changing the dates and searching for other things later. My one concern is a bug I ran into while utilizing componentDidMount() in another branch. When the app re-rendered it would re-run componentDidMount (? debatable?) and sort of lock the map to the current location.